_authentication.md

```diff
@@ -83,6 +85,55 @@ Once you have obtained an access token you may call any protected API method on
   This prevents attackers from stealing tokens from the user's browser history, logs, referer headers and other unsecure locations when credentials are sent via query URLs.
 </aside>
 
+## Client Credentials Flow
+Ideal for backend integrations that do not require access to other Uphold user accounts.
+
+For **business usage only** you may choose to use client credentials authentication. This requires manual approval from Uphold.
+
+### Creating a Token
+> To create a client credentials token, execute the following command (make sure the application is set to use client credentials and not authorization code):
+
+```bash
+curl https://api.uphold.com/oauth2/token \
+  -X POST \
+  -H "Content-Type: application/x-www-form-urlencoded" \
+  -u <clientId>:<clientSecret> \
+  -d 'grant_type=client_credentials'
+```
+
+To create a client credentials token you may use the following endpoint:
+
+`POST https://api.uphold.com/oauth2/token`
+
+Supported parameters:
+
+Parameter     | Required | Description
+------------- | -------- | -------------------------------------------------------------------------------------
+client_id     | yes      | The application's *clientId*. Please use HTTP Basic Authentication when possible.
+client_secret | yes      | The application's *clientSecret*. Please use HTTP Basic Authentication when possible.
+grant_type    | yes      | Must be set to *'client_credentials'*.
+
+<aside class="notice">
+  <strong>Important Notice</strong>: We recommend encoding the <i>clientId</i> and <i>clientSecret</i> with the HTTP Basic Authentication scheme, instead of authenticating via the request body.
+</aside>
+
+### Using the Token
+> Request using the 'Authorization header':
+
+```bash
+curl https://api.uphold.com/v0/me/cards \
+  -H "Authorization: Bearer <token>"
+```
+
+Once you have obtained a client credentials token you may call any protected API method on behalf of the user account owner of the application using the "Authorization" HTTP header in the format:
+
+`Authorization: Bearer <token>`
+<aside class="notice">
```

Comment:
I'd suggest adding a blank line before this one, to make the source code more legible (IIUC it has no effect in the final rendered text).